config/serverless.oblt.yml

```diff
@@ -27,3 +27,13 @@ xpack.apm.serverlessOnboarding: true
 xpack.fleet.packages:
   - name: apm
     version: latest
+## Disable APM UI components and API calls
+xpack.apm.featureFlags.agentConfigurationAvailable: false
+xpack.apm.featureFlags.configurableIndicesAvailable: false
+xpack.apm.featureFlags.infrastructureTabAvailable: false
+xpack.apm.featureFlags.infraUiAvailable: false
+xpack.apm.featureFlags.migrationToFleetAvailable: false
+xpack.apm.featureFlags.sourcemapApiAvailable: false
+xpack.apm.featureFlags.spacesAvailable: false
```

Comment:
Spaces plugin is available when the spaces plugin is enabled, and not available when it's disabled. I don't think we need a feature flag for this.

Reply:
Right, we are not using this feature flag

Reply:
Remove it here https://github.com/elastic/kibana/pull/159136/commits/33ad69ee5bcb488ce7d20194c5262873670d9146